refarch-integrations/refarch-s3-integration/refarch-s3-integration-core/src/test/java/de/muenchen/refarch/integration/s3/application/usecase/FileOperationsPresignedUrlUseCaseTest.java

```diff
@@ -26,34 +26,35 @@
 @MockitoSettings(strictness = Strictness.LENIENT)
 class FileOperationsPresignedUrlUseCaseTest {
 
+    public static final String TEST_TXT_PATH = "folder/test.txt";
+    public static final String EXAMPLE_PRESIGNED_URL = "some-presigned-url";
+    public static final String PATH_TO_FOLDER = "folder";
     @Mock
     private S3Adapter s3Adapter;
 
     private FileOperationsPresignedUrlInPort fileOperations;
 
     @BeforeEach
-    public void beforeEach() {
+    void beforeEach() {
         this.fileOperations = new FileOperationsPresignedUrlUseCase(this.s3Adapter);
         Mockito.reset(this.s3Adapter);
     }
 
     @Test
-    void getPresignedUrl() {
-        final String pathToFile = "folder/test.txt";
+    @SuppressWarnings("PMD.UnitTestShouldIncludeAssert")
```

Comment:
This supression should not be necessary.

Reply:
It is because pmd doesn't detect assert in the forEach. Don't know why i didn't but i just refactored it to a normal for loop.